# バンパセンサ(SCIBOT-Xのみ)GET
---
uint32 Bumper1
uint32 Bumper2
uint32 Bumper3
uint32 Bumper4

